packages/app/src/cli/models/extensions/specifications/flow_action.ts

```diff
@@ -37,7 +37,7 @@ export const FlowActionExtensionSchema = BaseSchemaWithHandle.extend({
 /**
  * Loads the schema from the partner defined file.
  */
-const loadSchemaPatchFromPath = async (extensionPath: string, patchPath: string | undefined) => {
+export const loadSchemaPatchFromPath = async (extensionPath: string, patchPath: string | undefined) => {
```

Comment:
I think there's a util folder we can move this function to `packages/app/src/cli/services/flow/..`. Can we move it there and write a test for this?